### DISCUSSION TITLE
How can I set up pages the correct way?

### DISCUSSION THREAD

**Discussion by ruru-m07** [fuma-nama/fumadocs#597] [Q&A] [answered]
## How can I set up pages the correct way? 

![image](https://github.com/user-attachments/assets/002cac65-c721-4bf4-8e30-5563f8c82f9b)

I try to read docs but I can't find the exact solution. : (

here I provide my `meta.json` file if needed.

```json
{
  "root": true,
  "pages": [
    "---Getting Started---",
    "index",
    "comparisons",
    "...",
    "---Components---",
    "...components",
    "---Others---",
    "[Github](https://github.com/ruru-m07/ruru-ui/)"
  ]
}
```

**Comment by fuma-nama** [comment, marked as answer]
Hmm could you elaborate more? If you only want to order the pages in the same way you've painted on the picture, it's also documented here: https://fumadocs.vercel.app/docs/headless/page-conventions#meta
Add the name of page to `pages` and it should be ordered as you want

**Reply by ruru-m07** [reply]
Thanks @fuma-nama ❤️ now I got it   :D

I do this:-

```json
{
  "root": true,
  "pages": [
    "---Getting Started---",
    "index",
    "installation",
    "components-json",
    "theming",
    "dark-mode",
    "cli",
    "typography",
    "changelog",
    "---Components---",
    "...components",
    "---Others---",
    "[Github](https://github.com/ruru-m07/ruru-ui/)"
  ]
}
```
